library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
-
-    * Non-biological-artifact <nowiki> {suggestedTag=Artifact-significance-to-recording} [Non-biological artifacts refer to anomalies in recordings introduced by external factors, potentially impacting the interpretation of the data.]</nowiki>
-        ** Power-supply-artifact <nowiki>[50-60 Hz artifact. Monomorphic waveform due to 50 or 60 Hz A/C power supply.]</nowiki>
-        ** Induction-artifact <nowiki>[Artifacts (usually of high frequency) induced by nearby equipment (like in the intensive care unit).]</nowiki>
-        ** Dialysis-artifact 
-        ** Artificial-ventilation-artifact 
-        ** Electrode-pops-artifact <nowiki>[Are brief discharges with a very steep upslope and shallow fall that occur in all leads which include that electrode.]</nowiki>
-        ** Salt-bridge-artifact <nowiki>[Typically occurs in 1 channel which may appear isoelectric. Only seen in bipolar montage.]</nowiki>
-
-    * Other-artifact <nowiki> {requireChild, suggestedTag=Artifact-significance-to-recording} </nowiki>
-        ** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-
-
-'''Polygraphic-channel-finding''' <nowiki>[Changes observed in polygraphic channels can be scored: EOG, Respiration, ECG, EMG, other polygraphic channel (+ free text), and their significance logged (normal, abnormal, no definite abnormality).]</nowiki>
+'''Polygraphic-channel-finding''' <nowiki>[Changes observed in polygraphic channels can be scored: EOG, Respiration, ECG, EMG, other polygraphic channel (+ free text), and their significance logged (normal, abnormal, no definite abnormality). (Source: Beniczky ea 2017, Section 13.)]</nowiki>
 
-    * EOG-channel-finding <nowiki> {suggestedTag=Finding-significance-to-recording} [EOG channel findings refer to the observations and characteristics associated with the recording of electrooculographic signals.]</nowiki>
+    * EOG-channel-finding <nowiki> {suggestedTag=Finding-significance-to-recording} [Electrooculogram (EOG) channel findings. (Source: Beniczky ea 2017, Section 13.)]</nowiki>
         ** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-    * Respiration-channel-finding <nowiki> {suggestedTag=Finding-significance-to-recording} [Respiration channel findings involve the observations and characteristics associated with the recording of respiratory signals.]</nowiki>
-        ** Respiration-oxygen-saturation {requireChild}
+    * Respiration-channel-finding <nowiki> {suggestedTag=Finding-significance-to-recording} [Findings in respiration sensors. (Source: Beniczky ea 2017, Section 13, Table 16.)]</nowiki>
+        ** Oxygen-saturation <nowiki> {requireChild}[Percentage + free text. (Source: Beniczky ea 2013, Table 9; Beniczky ea 2017, Table 16.)] </nowiki>
```

Comment:
Remove "Percentage + free text" from Oxygen-saturation

Reply:
Fixed to Percentage only: free text descriptions would render this non-machine readable and can be added in separate HED tags.